models/base_model.py

```diff
@@ -363,6 +376,67 @@ def get_current_fids(self):
                 fids[name] = float(getattr(self, name))  # float(...) works for both scalar tensor and float number
         return fids
 
+    def compute_D_accuracy_pred(self,real,fake,netD):
+        pred_real = netD(real).flatten()
+        pred_fake = netD(fake).flatten()
+        
+        FP=F.l1_loss(pred_fake, torch.zeros(pred_real.shape).to(self.device), reduction='sum')
+        TP=F.l1_loss(pred_real, torch.zeros(pred_real.shape).to(self.device), reduction='sum')
+        TN=F.l1_loss(pred_fake, torch.ones(pred_real.shape).to(self.device), reduction='sum')
+        FN=F.l1_loss(pred_real, torch.ones(pred_real.shape).to(self.device), reduction='sum')
```

Comment:
So it's not exactly the accuracy, since `pred_real` and `pred_fake` are not binary values but logits?